scripts/install.sh

```diff
@@ -30,63 +28,60 @@ svm_detect_profile() {
   DETECTED_PROFILE=''
 
   if [ -n "${BASH_VERSION-}" ]; then
-    if [ -f "$HOME/.bashrc" ]; then
-      DETECTED_PROFILE="$HOME/.bashrc"
-    elif [ -f "$HOME/.bash_profile" ]; then
-      DETECTED_PROFILE="$HOME/.bash_profile"
+    if [ -f "${HOME}/.bashrc" ]; then
+      DETECTED_PROFILE="${HOME}/.bashrc"
+    elif [ -f "${HOME}/.bash_profile" ]; then
+      DETECTED_PROFILE="${HOME}/.bash_profile"
     fi
   elif [ -n "${ZSH_VERSION-}" ]; then
-    DETECTED_PROFILE="$HOME/.zshrc"
+    DETECTED_PROFILE="${HOME}/.zshrc"
   fi
 
-  if [ -n "$DETECTED_PROFILE" ]; then
-    echo "$DETECTED_PROFILE"
+  if [ -n "${DETECTED_PROFILE}" ]; then
+    echo "${DETECTED_PROFILE}"
   fi
 }
 
 svm_install_dir() {
-  if [ -n "$SVM_DIR" ]; then
+  if [ -n "${SVM_DIR}" ]; then
     printf %s "${SVM_DIR}"
   else
     printf %s ${HOME}/.svm
   fi
 }
 
-svm_clone ()
-{
+svm_clone() {
   local INSTALL_DIR
-  INSTALL_DIR="$HOME/.svm"
+  INSTALL_DIR="${HOME}/.svm"
 
-  if [ -f "$INSTALL_DIR/sonic.go" ]; then
-    echo "=> svm is already installed in $INSTALL_DIR, trying to update the script"
+  if [ -f "${INSTALL_DIR}/sonic.go" ]; then
+    echo "=> svm is already installed in ${INSTALL_DIR}, trying to update the script"
     cd ~/.svm
     git pull origin master
   else
-    echo "=> Cloning svm as script to '$INSTALL_DIR'"
-    git clone --quiet https://github.com/openware/sonic.git  ~/.svm
+    echo "=> Cloning svm as script to '${INSTALL_DIR}'"
+    git clone --quiet https://github.com/openware/sonic.git ~/.svm
```

Comment:
It should be `git clone --quiet https://github.com/openware/sonic.git $INSTALL_DIR` I think